.github/workflows/os-increment-plugin-versions.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           - '2.8'
           - main
           - '2.9'
+          - 1.x
```

Comment:
@prudhvigodithi  AFAIK we do not want to add 1.x?

Reply:
Ya Sayali the automation adds it back, it would check for a version `.x` branch if missing.
https://github.com/opensearch-project/opensearch-build/blob/main/src/manifests_workflow/input_manifests.py#L210

Reply:
Once the 1.3.x is EOL, meaning no new versions of 1.3.x are incremented, then upon removal of `1.x` wont re-add back Sayali.

Reply:
Yeah let me remove it from this PR. You can review then.